client/src/components/Topic.jsx

```diff
@@ -41,8 +42,8 @@ export function Topic({ topic, responseOwner, submitButton = true }) {
       console.log('Topic response submitted');
       player.stage.set('submit', true);
     } else {
-      // should this instead be an Empirica Alert?
-      alert('Please select an answer before continuing');
+      // TODO: make this an Empirica Alert
```

Comment:
Is this still a todo? I think you implemented the alert below?

Reply:
good catch. updated.